07/hmw-07.py

```diff
@@ -0,0 +1,86 @@
+phone_book = dict()
+
+# create the initial phone book
```

Comment:
For the future: there is no need to keep unused code commented. You can delete it. If you need this code as an example, you can keep it in some other file and don't commit that file